.gitignore

```diff
@@ -2,7 +2,6 @@
 Make.dep
 .built
 *.o
-*.a
```

Comment:
We can't remove this from top of .gitignore. If you want to add your library on your folder, add new .gitignore in yours and make it included.